zcash_primitives/src/transaction/builder.rs

```diff
@@ -66,6 +66,15 @@ pub enum Error<FeeError> {
     TransparentBuild(transparent::builder::Error),
     /// An error occurred in constructing the Sapling parts of a transaction.
     SaplingBuild(sapling_builder::Error),
+    /// An error occurred in constructing the Orchard parts of a transaction.
+    OrchardBuild(orchard::builder::BuildError),
+    /// An error occurred in adding an Orchard Spend a transaction.
+    OrchardSpend(orchard::builder::SpendError),
+    /// An error occurred in adding an Orchard Output a transaction.
+    OrchardRecipient(orchard::builder::OutputError),
+    /// Orchard is not yet active on the network, and the user attempted to add
+    /// Orchard parts to a transaction.
```

Comment:
```suggestion
    /// The builder was constructed without an Orchard anchor, but an Orchard spend or
    /// recipient was added. This might be because Orchard is not yet active on the
    /// network.
```